charts/cronjob/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
@sre we should proceed with adding our helm charts to this repo https://github.com/hpi-schul-cloud/helm-charts, to avoid duplication